bootloader/bootloader.go

```diff
@@ -234,9 +234,18 @@ func InstallBootConfig(gadgetDir, rootDir string, opts *Options) error {
 	return fmt.Errorf("cannot find boot config in %q", gadgetDir)
 }
 
+type bootloaderNewFunc func(rootdir string, opts *Options) Bootloader
+
 var (
 	forcedBootloader Bootloader
 	forcedError      error
+
+	bootloaders = []bootloaderNewFunc{
```

Comment:
I would  give this its own var close to the type definition of the function with a comment like

// bootloaders list all possible bootloaders by their constructor function.